scripts/lib.sh

```diff
@@ -6,6 +6,52 @@ require() {
     command -v "$1" >/dev/null 2>&1
 }
 
+sudo_askpass() {
+  if [ -z "${SUDO_ASKPASS-x}" ]; then
+    sudo --askpass "$@"
+  else
+    sudo "$@"
+  fi
+}
+
+# After using homebrew to install docker, we need to do some magic to remove the need to interact with the GUI
+# See: https://github.com/docker/for-mac/issues/2359#issuecomment-607154849 for why we need to do things below
+init_docker() {
+  # Need to start docker if it was freshly installed (docker server is not running)
+  if ! command -v docker &>/dev/null && [ -d "/Applications/Docker.app" ]; then
+    echo "Making some changes to complete Docker initialization"
+    # allow the app to run without confirmation
+    xattr -d -r com.apple.quarantine /Applications/Docker.app
+
+    # preemptively do docker.app's setup to avoid any gui prompts
+    sudo_askpass /bin/mkdir -p /Library/PrivilegedHelperTools
+    sudo_askpass /bin/chmod 754 /Library/PrivilegedHelperTools
+    sudo_askpass /bin/cp /Applications/Docker.app/Contents/Library/LaunchServices/com.docker.vmnetd /Library/PrivilegedHelperTools/
+    sudo_askpass /bin/cp /Applications/Docker.app/Contents/Resources/com.docker.vmnetd.plist /Library/LaunchDaemons/
+    sudo_askpass /bin/chmod 544 /Library/PrivilegedHelperTools/com.docker.vmnetd
+    sudo_askpass /bin/chmod 644 /Library/LaunchDaemons/com.docker.vmnetd.plist
+    sudo_askpass /bin/launchctl load /Library/LaunchDaemons/com.docker.vmnetd.plist
+  fi
+  start_docker
+}
+
+start_docker() {
```

Comment:
This function and the ones above come from:
https://github.com/getsentry/bootstrap-sentry/blob/dcf3f9b8ac3a6e40b0dce5fc72ad2c56789af21d/bootstrap.sh#L347-L376
https://github.com/getsentry/bootstrap-sentry/blob/dcf3f9b8ac3a6e40b0dce5fc72ad2c56789af21d/bootstrap.sh#L88-L94